src/app/components/LanguageInfo.svelte

```diff
@@ -118,7 +134,15 @@
       cyData="input-copyright"
       subtext="" />
   </div>
-
+  <div class="copyright-field">
+    <Button
+      size="small"
+      color="blue"
+      isOutlined={true}
+      onClick={addCopyrightSymbol}>
+      add ©
```

Comment:
nit: We might want to make this `add ©` value use i18n. Here's the relevant PR for localization: #279

Reply:
Added